src/plugins/joinAndQuit/plugin.ts

```diff
@@ -154,7 +155,7 @@ export class JoinAndQuit extends ProposalPlugin<
     let opts
     if ((await state).pluginParams.fundingToken === NULL_ADDRESS) {
       // if we have no funding token, we shoudl send the fee as ETH
-      opts = { value: options.fee.toNumber()}
+      opts = { value: new BigNumber(options.fee.toString()).toHexString() }
```

Comment:
I have nothing against this if this works, but in other places we just pass a a string to the contracts (instead of a BigNumber), so this could just read "options.fee.toString()"

Reply:
Both changes are depended, and if I change here to "toString" I must change in `operation.ts` to "`string`".
In other places we pass also numbers and there is no reason that `fee` would not be a number also.

Reply:
fee can a `String` or `BN` .
for sure it cannot be `Number`  as it not fits in 53 bits 
I think as it is with the update it will cover all cases... both passing `BN` or `String`

Reply:
> I have nothing against this if this works, but in other places we just pass a a string to the contracts (instead of a BigNumber), so this could just read "options.fee.toString()"

though the test pass a BN